wherehows-web/app/routes/datasets/dataset.js

```diff
@@ -319,6 +323,33 @@ export default Route.extend({
         return Promise.reject(new Error('Dataset references request failed.'));
       })
       .catch(() => set(controller, 'hasReferences', false));
+
+    // TODO: Get stutus current user to see data in ACL access tab
+    Promise.resolve(currentUser())
+      .then(userInfo => {
+        setProperties(controller, {
+          userInfo
+        });
+        return checkAclAccess(userInfo.userName).then(value => {
+          setProperties(controller, {
+            aclAccessResponse: value,
+            currentUserInfo: userInfo.userName,
+            aclUsers: value.body
+          });
+        });
+      })
+      // return Promise.resolve(checkAclAccess(userInfo.userName)).then(value => {
```

Comment:
Let's not leave commented out code in the codebase, if you want to retain, preferably use functionality provided by git instead, e.g. `git stash`